spark/src/main/java/org/opensearch/sql/spark/dispatcher/model/FullyQualifiedTableName.java

```diff
@@ -5,6 +5,9 @@
 
 package org.opensearch.sql.spark.dispatcher.model;
 
+import static org.apache.commons.lang3.StringUtils.strip;
+import static org.opensearch.sql.spark.dispatcher.model.IndexQueryDetails.STRIP_CHARS;
```

Comment:
Nit: Should we bring this constant to this File.

Reply:
sure, will merge this first. CI took so long every commit ...